config/routes.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   namespace :users do
-    resources :applications do
+    resources :service_providers do
     end
   end
 
```

Comment:
good point. My intention was to eventually add `Group` or `Org` in order to define authz for a group of `User`s.